vantage6/common/docker_addons.py

```diff
@@ -291,10 +292,27 @@ def remove_container(container: Container, kill=False) -> None:
     if kill:
         try:
             container.kill()
-        except Exception as e:
+        except Exception:
             pass  # allow failure here, maybe container had already exited
     try:
         container.remove()
     except Exception as e:
         log.error(f"Failed to remove container {container.name}")
         log.debug(e)
+
+
+def get_server_config_name(container_name: str, scope: str):
+    """
+    Get the configuration name of a server from its docker container name
+
+    Docker container name of the server is formatted as
```

Comment:
Add input parameters in the Numpydoc format (for readthedocs)

Reply:
Added